### ISSUE TITLE
Options validation doesn't work for Windows modules

### ISSUE THREAD

**Issue by nre-ableton** [ansible/ansible-lint#2831]
##### Summary

In ansible-lint 6.10.0, a new options validation rule was introduced (see https://github.com/ansible/ansible-lint/pull/2749). However, this doesn't seem to work for Windows modules, and an error message is emitted to the log when ansible-lint fails to load the module, similar to the following:

> `WARNING  Unable to load module ansible.windows.win_regedit at ansible/roles/<redacted>/<redacted.yml>:2 for options validation`

We are observing this error for both modules in the `ansible.windows` namespace, but also `community.windows`.

**Comment by ssbarnea** [comment]
Not a bug, please use forum to ask for support. I tested myself on macos and modules are found after I installed `community.windows`.

**Comment by bluikko** [comment]
@ssbarnea And did you test offline mode as well?

**Comment by redtux** [comment]
JFYI: I was having the same issue with `ansible-lint` 6.14.6 in a Python virtual environment.

Upgrading to 6.15.0 via pip (plus upgrading `ansible-core` from 2.14.4 to 2.14.5 and `ansible` from 7.4.0 to 7.5.0) helped to fix the issue.

- System: `Ubuntu 22.04.2 LTS`
- `ansible --version`

```yaml
ansible [core 2.14.5]
  config file = /home/redtux/git/myproject/ansible_windows/ansible.cfg
  configured module search path = ['/home/redtux/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/redtux/git/myproject/ansible_windows/.venv/lib/python3.10/site-packages/ansible
  ansible collection location = /home/redtux/.ansible/collections:/usr/share/ansible/collections
  executable location = /home/redtux/git/myproject/ansible_windows/.venv/bin/ansible
  python version = 3.10.6 (main, Mar 10 2023, 10:55:28) [GCC 11.3.0] (/home/redtux/git/myproject/ansible_windows/.venv/bin/python3)
  jinja version = 3.1.2
  libyaml = True

```
